src/context/AuthContext.jsx

```
@@ -0,0 +1,80 @@
+import React, { createContext, useContext, useState, useEffect } from 'react';
+
+/**
+ * Auth Context — menyediakan state autentikasi untuk seluruh app.
+ * Dipakai oleh Navbar (LOGIN vs PROFILE) dan tombol aksi di halaman detail.
+ *
+ * Struktur disepakati lintas tim (Orang 1, 2, 3, 4).
+ */
+const AuthContext = createContext({
+  isAuthenticated: false,
+  user: null,
+  login: (token, user) => {},
```

ESLint

[error] 'token' is defined but never used.

(no-unused-vars)